ast/spec.md

```diff
@@ -149,6 +151,15 @@ interface Identifier <: Expression, Pattern {
 }
 ```
 
+# PrivateName
+
+```js
+interface PrivateName <: Expression, Pattern {
```

Comment:
Apart from dan's comments about grammar, @loganfsmyth, I though it was cleaner (at least while is at Stage2) to make additive changes when possible rather than mutating existing nodes, like adding a `private` attribute for this case. We can recondile in the next mayor version, once public fields, private fields et al, are in +Stage3